### PR TITLE
Add slither analyse on Jenkins for all new concerned contracts

### PR DESCRIPTION
**Pull request by gfournieriExec**
Slither's analyse makes mention of : 
contracts/Store.v8.sol => 69 times 
contracts/modules/delegates/IexecPoco2Delegate.sol => 68
contracts/modules/delegates/IexecPoco1Delegate.so => 62
contracts/modules/delegates/IexecEscrow.v8.sol => 11
contracts/modules/delegates/SignatureVerifier.v8.sol => 8
contracts/modules/delegates/IexecPocoCommonDelegate.sol => 6
contracts/modules/DelegateBase.v8.sol => 1
contracts/modules/delegates/IexecPocoAccessorsDelegate.sol =>1

### CODE REVIEW

File: contracts/tools/testing/Slither.sol [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+pragma solidity ^0.8.0;
+
+import {IexecPocoAccessorsDelegate} from "../../modules/delegates/IexecPocoAccessorsDelegate.sol";
+import {IexecPoco1Delegate} from "../../modules/delegates/IexecPoco1Delegate.sol";
+import {IexecPoco2Delegate} from "../../modules/delegates/IexecPoco2Delegate.sol";
+
+/**
+ * @notice This contract is dedicated to slither testing.
+ */
+contract AnalyseSlither is IexecPocoAccessorsDelegate, IexecPoco1Delegate, IexecPoco2Delegate {}
```
Comment (jeremyjams) [inline contracts/tools/testing/Slither.sol]:
```suggestion
contract Slither is IexecPocoAccessorsDelegate, IexecPoco1Delegate, IexecPoco2Delegate {}
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## vNEXT
+- Create slither smart contract entry point and run slither analyse on new contracts. (#87)
```
Comment (jeremyjams) [inline CHANGELOG.md:4]:
```suggestion
- Create slither smart contract entry point and run slither analysis on new contracts. (#87)
```

File: contracts/tools/testing/Slither.sol [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.0;
+
+import {IexecPocoAccessorsDelegate} from "../../modules/delegates/IexecPocoAccessorsDelegate.sol";
+import {IexecPoco1Delegate} from "../../modules/delegates/IexecPoco1Delegate.sol";
+import {IexecPoco2Delegate} from "../../modules/delegates/IexecPoco2Delegate.sol";
+
+/**
+ * @notice This contract is dedicated to slither testing.
```
Comment (jeremyjams) [inline contracts/tools/testing/Slither.sol]:
```suggestion
 * @notice This contract is dedicated to slither analysis.
```

File: slither.config.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
     "solc_remaps": "@=node_modules/@",
-    "filter_paths": "node_modules",
+    "filter_paths": "(node_modules/|contracts/modules/interfaces/|contracts/libs/|contracts/external/|contracts/registries/)",
```
Comment (jeremyjams) [inline slither.config.json:3]: Does scanning `contracts/libs` & `contracts/registries` creates issues (because of solidity version)?

Comment (gfournieriExec) [reply slither.config.json:3]: It does not. We can add it, for the moment most of the alerts are related to pragma version / mixedCase so could be ignored in the slither.config.json with  `detectors_to_exclude`

File: contracts/tools/testing/slither/Slither.sol [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.0;
+
+import {IexecPocoAccessorsDelegate} from "../../../modules/delegates/IexecPocoAccessorsDelegate.sol";
+import {IexecPoco1Delegate} from "../../../modules/delegates/IexecPoco1Delegate.sol";
+import {IexecPoco2Delegate} from "../../../modules/delegates/IexecPoco2Delegate.sol";
+
+/**
+ * @notice This contract is dedicated to slither analysis.
```
Comment (zguesmi) [inline contracts/tools/testing/slither/Slither.sol:11]: Can you add here the reason why it was created, please.

**Review by jeremyjams — APPROVED** [review]
Thanks :)

File: contracts/tools/testing/slither/Slither.sol (hunk 5 above)
Comment (gfournieriExec) [reply contracts/tools/testing/slither/Slither.sol:11]: Done see : [update notices](https://github.com/iExecBlockchainComputing/PoCo/pull/87/commits/d2594ba5fa7c3653c5fe101697cf05a8125e3293)

Comment (zguesmi) [reply contracts/tools/testing/slither/Slither.sol:11]: Thanks!

**Review by zguesmi — APPROVED** [review]

**Pull request merged**